data_models/website.py

```diff
@@ -14,7 +14,7 @@ class Website:
 
     def __init__(self, url):
         self.url = url_utils.valid_url(url)
-        self.is_valid = True if url else False
+        self.is_valid = True if self.url else False
```

Comment:
data_models/website.py:19:4: C0116: Missing function or method docstring (missing-function-docstring)
data_models/website.py:32:-1: W0105: String statement has no effect (pointless-string-statement)
data_models/website.py:39:4: C0116: Missing function or method docstring (missing-function-docstring)

Reply:
My bad! Forgot to do it for data models.